Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -335,7 +337,7 @@ public PresignedPostPolicyArgs WithPolicy(PostPolicy policy)
         {
             this.Policy = policy;
             return this;
-        }
+        }    
```

Comment:
why is there an extra space

Reply:
Happened during vscode 'Accept Both changes'. Removing.